bofire/surrogates/diagnostics.py

```diff
@@ -485,18 +486,24 @@ def validate_shapes(cls, values):
                 )
         return values
 
+    # TODO[pydantic]: We couldn't refactor the `validator`, please replace it by `field_validator` manually.
```

Comment:
Does `TODO[pydantic]` have a special meaning? And who is the one to replace stuff manually?

Reply:
Very good catch, i fixed it.